diff --git a/node_modules/react-tippy/index.d.ts b/node_modules/react-tippy/index.d.ts
index a95c697..68585d9 100644
--- a/node_modules/react-tippy/index.d.ts
+++ b/node_modules/react-tippy/index.d.ts
@@ -51,6 +51,8 @@ export interface TooltipProps {
   sticky?: boolean;
   stickyDuration?: boolean;
   beforeShown?: () => void;
+  onShow?: () => void;
+  onHide?: () => void;
   shown?: () => void;
   beforeHidden?: () => void;
   hidden?: () => void;
